internal/crypto/src/openssl/validators/mod.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License,
+// Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// or the MIT license (http://opensource.org/licenses/MIT),
+// at your option.
+
+// Unless required by applicable law or agreed to in writing,
+// this software is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or
+// implied. See the LICENSE-MIT and LICENSE-APACHE files for the
+// specific language governing permissions and limitations under
+// each license.
+
+//! This module binds OpenSSL logic for validating raw signatures to this
+//! crate's [`RawSignatureValidator`] trait.
+
+use crate::{raw_signature::RawSignatureValidator, SigningAlg};
+
+mod ecdsa_validator;
+pub use ecdsa_validator::EcdsaValidator;
+
+mod ed25519_validator;
+pub use ed25519_validator::Ed25519Validator;
+
+// mod rsa_validator;
+// pub use rsa_validator::{RsaLegacyValidator, RsaValidator}; // ???
```

Comment:
Why the `???`?

Reply:
I'm not sure if `RsaLegacyValidator` needs to be made public or brought over.